src/EventServiceProviderTrait.php

```diff
@@ -52,7 +52,13 @@ public function testListenerMapEventsExist()
         $map = $this->getListenerMap();
 
         foreach (array_keys($map) as $event) {
-            $this->assertTrue(class_exists($event));
+            $reflection = new ReflectionClass($event);
+
+            if ($reflection->isInterface()) {
+                $this->assertTrue(interface_exists($event));
```

Comment:
No need to assert this. Just ask the reflection class if is not trait. If it didn't exist, then creating a new reflection class would have failed actually.